src/components/AuthDialog/index.tsx

```diff
@@ -0,0 +1,65 @@
+'use client'
+
+import Image from 'next/image'
+import * as React from 'react'
+import { useState } from 'react'
+
+import { Button } from '@/components/ui/button'
+import { Dialog, DialogContent, DialogTrigger } from '@/components/ui/dialog'
+
+import ForgotPassword from './ForgotPassword'
+import Login from './login'
+import SignUp from './SignUp/SignUp'
```

Comment:
I think the file inside the SignUp directory should be named index

Reply:
[74810f1](https://github.com/dark-orga/anitube-website/pull/2/commits/74810f1ca9ded7c30bb6f4f2e3fb8e9be2f46431)